### PR TITLE
offer scheduler added

### PR DESCRIPTION
**Pull request by jabajke**

### CODE REVIEW

File: customer/tasks.py [hunk 1]
```diff
@@ -1,5 +1,56 @@
...
-    pass
+    cheapest_cars = []
+    for offer in Offer.objects.filter(is_active=True):
+        suitable_autohouse_car = AutohouseCar.objects.filter(
+            **offer.update_keys('car__'),
+            price__lte=offer.price,
+            is_active=True
+        )
+        try:
+            min_default_price = suitable_autohouse_car.order_by('price')[0]
```
Comment (shprexen) [inline customer/tasks.py:27]: let's use .first() method instead [0] and then check if such car exists, if it is not - we should keep somewhere information, that offer hasn't completed

File: customer/tasks.py [hunk 2]
```diff
@@ -1,5 +1,56 @@
...
+            autohouse_car__price__lte=offer.price,
+            is_active=True
+        )
+        try:
+            min_discount_price = (
+                discount_cars
+                .annotate(
+                    price=(F('autohouse_car__price') * ((100 - F('discount')) / 100))
+                )
+                .order_by('price'))[0]
```
Comment (shprexen) [inline customer/tasks.py:42]: the same, use .first method

**Review by shprexen — CHANGES_REQUESTED** [review]

File: customer/tasks.py (hunk 2 above)
Comment (jabajke) [reply customer/tasks.py]: done

File: customer/tasks.py (hunk 1 above)
Comment (jabajke) [reply customer/tasks.py]: done

**Review by shprexen — APPROVED** [review]

**Pull request merged**